config/default.json

```diff
@@ -0,0 +1,14 @@
+{
+    "penalties" : [
+        "LACKING_NURSE",
+        "LACKING_WORKER",
```

Comment:
Pytanie czy nie rozdzielić dzień noc tych brakujących workerów, nie wiem. Co myślisz?

Reply:
Dla obu typów? No można w sumie, większa funkcjonalność, a będzie można najwyżej to wyłączyć zrównując je.

Reply:
Tylko dla` "LACKING_WORKER"`, bo to będzie róźnica jak się zmieni kiedy dzień się zaczyna, a kiedy noc